From f128e5cccc856f6825c032d1191d9ca08f3b2c03 Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Mon, 19 Mar 2018 17:42:57 +0800
Subject: [PATCH 3526/5242] MLK-17837-02 dts: imx7ulp-evk: add rpmsg sensor
 support

commit  343f14f15cc85c722ceaf1bdba8314aa1fd19aa8 from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable rpmsg input device like sensor support for i.MX7ULP B0
EVK board.

Reviewed-by: Elven Wang <elven.wang@nxp.com>
Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 37e28ea..197634d 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -189,6 +189,10 @@
 		};
 	};
 
+	rpmsg_sensor: rpmsg-sensor {
+		compatible = "fsl,rpmsg-input";
+	};
+
 	sound-rpmsg {
 		compatible = "fsl,imx-audio-rpmsg";
 		model = "rpmsg-audio";
@@ -493,7 +497,7 @@
 &rpmsg{
 	/*
 	 * 64K for one rpmsg instance, default using 2 rpmsg instances:
-	 * --0x9FF00000~0x9FF0FFFF: pmic,pm,audio,keys,gpio
+	 * --0x9FF00000~0x9FF0FFFF: pmic,pm,audio,keys,gpio,sensor
 	 * --0x9FF10000~0x9FF1FFFF: pingpong,virtual tty
 	 */
 	vdev-nums = <2>;
-- 
1.7.9.5

